tests/test_opds.py

```diff
@@ -1742,6 +1742,60 @@ def test_format_types(self):
         eq_([OPDSFeed.ENTRY_TYPE, Representation.TEXT_HTML_MEDIA_TYPE + DeliveryMechanism.STREAMING_PROFILE],
             AcquisitionFeed.format_types(overdrive_streaming_text))
 
+    def test_add_breadcrumbs(self):
+        # Testing not passing an entrypoint when creating the breadcrumbs
+        no_entrypoint_wl = WorkList()
+        no_entrypoint_wl.initialize(
+            library=self._default_library, display_name="no_entrypoint"
+        )
+        no_entrypoint_wl.lane = self._lane()
+
+        feed_no_eps = AcquisitionFeed(
+            self._db, self._str, self._url, [], annotator=TestAnnotator
+        )
+        feed_no_eps.add_breadcrumbs(no_entrypoint_wl.lane)
+        feed = feedparser.parse(unicode(feed_no_eps))['feed']
+
+        eq_(len(feed.links), 2)
+        eq_(feed.links[1].href, TestAnnotator.default_lane_url())
+        eq_(feed.links[1].title, TestAnnotator.top_level_title())
+
+        # Passing ebook entrypoint to a single lane feed
+        entrypoints = [AudiobooksEntryPoint, EbooksEntryPoint]
+        with_entrypoints_wl = WorkList()
+        with_entrypoints_wl.initialize(library=self._default_library, display_name="wl",
+                           entrypoints=entrypoints)
+        with_entrypoints_wl.lane = self._lane()
+
+        single_lane_feed_with_eps = AcquisitionFeed(
+            self._db, self._str, self._url, [], annotator=TestAnnotator
+        )
+        single_lane_feed_with_eps.add_breadcrumbs(with_entrypoints_wl.lane, entrypoint=EbooksEntryPoint)
```

Comment:
I'm not sure if I'm doing this correctly, but with how I'm calling the `add_breadcrumbs` functions to test it, it doesn't seem like I need to create a `WorkList`. It looks like I can pass in `self._lane()` into this function instead of a lane from a `WorkList`.

Reply:
Yes, I think this is fine.